src/main/frontend/util/marker.cljs

```diff
@@ -61,21 +49,16 @@
     (if (= :now preferred-workflow) "LATER" "TODO")))
 
 (defn cycle-marker
-  [content format preferred-workflow]
-  (let [markdown? (= :markdown format)
-        match-fn (fn [marker] (util/safe-re-find (header-marker-pattern markdown? marker)
-                                                content))]
-    (cond
-     (match-fn "TODO")
-     [(replace-marker content markdown? "TODO" "DOING") "DOING"]
-     (match-fn "DOING")
-     [(replace-marker content markdown? "DOING" "DONE") "DONE"]
-     (match-fn "LATER")
-     [(replace-marker content markdown? "LATER" "NOW") "NOW"]
-     (match-fn "NOW")
-     [(replace-marker content markdown? "NOW" "DONE") "DONE"]
-     (match-fn "DONE")
-     [(replace-marker content markdown? "DONE" "") nil]
-     :else
-     (let [marker (if (= :now preferred-workflow) "LATER" "TODO")]
-       [(add-or-update-marker (string/triml content) format marker) marker]))))
+  "The cycle-marker will cycle markers sequentially. You can find all its order in `cycle-marker-state`.
+
+  It also accept the specified `marker` and `new-marker`.
```

Comment:
typo here: accepts.

Reply:
Got it! Fixed in [113ae7e](https://github.com/logseq/logseq/pull/4661/commits/113ae7e687a942c4eb426c22c82c088768e6dcb6).